service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PbaNumberHelper.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.model.common.C2DocumentBundle;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class PbaNumberHelper {
+
+    private PbaNumberHelper() {
+    }
```

Comment:
`@NoArgsConstructor(access = AccessLevel.PRIVATE)` 👈 is this better?

Reply:
no difference, going to leave it as it is because it's the way we do it in every helper